config/routes.rb

```diff
@@ -1,4 +1,4 @@
 Rails.application.routes.draw do
-  devise_for :users
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :listings
```

Comment:
what's listings?

Reply:
My bad. It was an entity I created for testing.